src/sage/modular/modform/element.py

```diff
@@ -3544,18 +3544,30 @@ def _mul_(self, other):
         TESTS::
 
             sage: M = ModularFormsRing(1)
-            sage: f4 = ModularForms(1, 4).0; f6 = ModularForms(1, 6).0;
-            sage: F4 = M(f4); F6 = M(f6);
+            sage: F4 = M.0; F6 = M.1;
             sage: F4*F6 # indirect doctest
             1 - 264*q - 135432*q^2 - 5196576*q^3 - 69341448*q^4 - 515625264*q^5 + O(q^6)
+
+        This shows that the issue at :trac:`35932` is fixed::
```

Comment:
"trac" refers to the old issues tracker of SageMath. The following should work:
```suggestion
        This shows that the issue at :issue:`35932` is fixed::
```